### PR TITLE
ADD: eccodes.xmod & instructions on how to get .xmod files in README.md

### PR DESCRIPTION
**Pull request by elsagermann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,12 @@
 [![Build Status](https://travis-ci.org/claw-project/omni-xmod-pool.svg?branch=master)](https://travis-ci.org/claw-project/omni-xmod-pool)
 
 Sets of generic xmod file for OMNI Compiler
+
+## Building .xmod files using F_Front
+
+Use the F_Front binary of the claw installation and a Fortran file in order to create a .xmod file. **Careful:** if your file depends on other modules, they also have to be transformed into .xmod before being
```
Comment (clementval) [inline README.md]:
```suggestion
Use the `F_Front` binary of the claw installation and a Fortran file in order to create a `.xmod` file. **Careful:** if your file depends on other modules, they also have to be transformed into `.xmod` before being 
```

File: README.md [hunk 2]
```diff
@@ -3,3 +3,12 @@
 [![Build Status](https://travis-ci.org/claw-project/omni-xmod-pool.svg?branch=master)](https://travis-ci.org/claw-project/omni-xmod-pool)
 
 Sets of generic xmod file for OMNI Compiler
+
+## Building .xmod files using F_Front
+
+Use the F_Front binary of the claw installation and a Fortran file in order to create a .xmod file. **Careful:** if your file depends on other modules, they also have to be transformed into .xmod before being 
+able to be read by the F_Front! Include module dependencies using the -M option and -I option for headerdependencies:
+
+    $ ./F_Front code.f90 -M [.xmod dirpath] -I [.h dirpath]
```
Comment (clementval) [inline README.md]:
```suggestion
able to be read by the `F_Front`! Include module dependencies using the `-M` option and `-I` option for header dependencies:

`$ ./F_Front code.f90 -M [.xmod dirpath] -I [.h dirpath]`

```

**Review by clementval — COMMENTED** [review]
I just formatted a bit the README changes.

**Comment by elsagermann** [comment]
Perfect, thanks!

**Pull request merged**